detox/src/artifacts/instruments/android/AndroidInstrumentsRecording.js

```diff
@@ -0,0 +1,22 @@
+const log = require('../../../utils/logger').child({ __filename });
+const InstrumentsArtifactRecording = require('../InstrumentsArtifactRecording');
+
+class AndroidInstrumentsRecording extends InstrumentsArtifactRecording {
+  constructor({ adb, client, deviceId, temporaryRecordingPath }) {
+    super({ client, temporaryRecordingPath });
+    this.adb = adb;
+    this.deviceId = deviceId;
+  }
+
+  async doSave(artifactPath) {
+    await super.doSave(artifactPath);
+    await this.adb.pull(this.deviceId, this.temporaryRecordingPath, artifactPath);
```

Comment:
Just musing, what will we do if we don't find a file there... ? If actual profiler code won't be there, I mean. @valentynberehovyi

Reply:
Should we hint as well like below... ?
```
SimulatorInstrumentsRecording.hintAboutDetoxInstruments = _.once(() => {
   log.warn(`Make sure either:
 1. You have installed Detox Instruments:
    https://github.com/wix/DetoxInstruments#installation 
 2. You have integrated Detox Instruments in your app:
    https://github.com/wix/DetoxInstruments/blob/master/Documentation/XcodeIntegrationGuide.md 
 3. You have set the environment variable with your custom Detox Instruments location:
    export DETOX_INSTRUMENTS_PATH="/path/to/Detox Instruments.app"`);
 });
```

Reply:
We'll add this later. For now recordings will not start when instruments is not installed.